### PR TITLE
Edpm networker node

### PR DESCRIPTION
**Pull request by slawqo**
This PR adds make file targets to create and deploy EDPM networker node(s) in very similar way to how edpm compute nodes are created.

Depends-On: https://github.com/openstack-k8s-operators/dataplane-operator/pull/590

### CODE REVIEW

**Review by fao89 — COMMENTED** [review]
I think we should extract common code into some `edpm_node` root, and `edpm_compute` /`edpm_networker` would call `edpm_node` passing the parameter compute/networker

**Comment by slawqo** [comment]
> I think we should extract common code into some `edpm_node` root, and `edpm_compute` /`edpm_networker` would call `edpm_node` passing the parameter compute/networker

I totally agree that we should reorganize it and extract some common code to reuse. But it's not just edpm-compute and now edpm-networker which are duplicating stuff like that. Please see e.g. scripts/gen-edpm-baremetal-kustomize.sh or devsetup/scripts/gen-edpm-compute-node-bgp.sh which are already there.
Would it be ok for You if we would go with this patch as it is now and I will propose new PR to clean this stuff up and reorganize code in the way that we will reuse as much as possible?

**Review by fao89 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -721,7 +722,9 @@ edpm_deploy_prep: export EDPM_NTP_SERVER=${DATAPLANE_NTP_SERVER}
 edpm_deploy_prep: export EDPM_REGISTRY_URL=${DATAPLANE_REGISTRY_URL}
 edpm_deploy_prep: export EDPM_CONTAINER_TAG=${DATAPLANE_CONTAINER_TAG}
 edpm_deploy_prep: export EDPM_CONTAINER_PREFIX=${DATAPLANE_CONTAINER_PREFIX}
-edpm_deploy_prep: export EDPM_EXTRA_NOVA_CONFIG_FILE=${DEPLOY_DIR}/25-nova-extra.conf
+edpm_deploy_prep: export EDPM_EXTRA_NOVA_CONFIG_FILE=${DEPLOY_DIR_EDPM_COMPUTE}/25-nova-extra.conf
+edpm_deploy_prep: export EDPM_DEPLOY_DIR=${DEPLOY_DIR_EDPM_COMPUTE}
+edpm_deploy_prep: export EDPM_SERVER_ROLE=compute
```
Comment (fao89) [inline Makefile:727]: sweet!

**Review by fao89 — APPROVED** [review]
Thank you!

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by slawqo** [comment]
I just tested this once again locally and after deploying 2 edpm compute nodes and 2 edpm networker nodes I have all of that working fine:

`sh-5.1$ openstack network agent list
+--------------------------------------+------------------------------+--------------------+-------------------+-------+-------+----------------------------+
| ID                                   | Agent Type                   | Host               | Availability Zone | Alive | State | Binary                     |
+--------------------------------------+------------------------------+--------------------+-------------------+-------+-------+----------------------------+
| 423849d5-7161-41f8-9bd4-1f9c1f2aa7c0 | OVN Controller Gateway agent | crc-mmwz7-master-0 |                   | :-)   | UP    | ovn-controller             |
| 9689a789-6932-4933-a62a-5e23fa92c440 | OVN Controller Gateway agent | edpm-networker-0   |                   | :-)   | UP    | ovn-controller             |
| addcaef5-e3c6-48bb-ba03-e691cb5ce8bc | OVN Controller Gateway agent | edpm-networker-1   |                   | :-)   | UP    | ovn-controller             |
| 6d08c23d-06dc-4570-9dc4-46299af86037 | OVN Controller agent         | edpm-compute-0     |                   | :-)   | UP    | ovn-controller             |
| 2d6d5c4d-de81-4da3-b085-9012f2f5ccef | OVN Controller agent         | edpm-compute-1     |                   | :-)   | UP    | ovn-controller             |
| 3d8561a3-235b-5e11-b358-d52161ccd621 | OVN Metadata agent           | edpm-compute-0     |                   | :-)   | UP    | neutron-ovn-metadata-agent |
| 7cdab5fe-33a3-54a3-834c-e4a60aafff99 | OVN Metadata agent           | edpm-compute-1     |                   | :-)   | UP    | neutron-ovn-metadata-agent |
+--------------------------------------+------------------------------+--------------------+-------------------+-------+-------+----------------------------+`

**Comment by slawqo** [comment]
recheck

**Pull request merged**